src/partials/contacts.html

```diff
@@ -1,48 +1,48 @@
 <section class="contacts">
   <div class="container">
-    <h2 class="contacts__main-header">Contacts</h2>
+    <h2 class="contacts__header">Contacts</h2>
     <ul class="contacts__list">
-      <li class="contacts__list-item">
+      <li class="contacts__item">
         <h3 class="contacts__label">Cafe</h3>
-        <h4 class="contacts__location">Chicago</h4>
+        <h4 class="contacts__title">Chicago</h4>
         <p class="contacts__text">Fusce ut velit laoreet, tempus arcu molestie vulputate</p>
-        <p class="contacts__days">Monday - Friday</p>
-        <time class="contacts__time">06:00 Am</time> - <time class="contacts__time">10:00 pm</time>
-        <p class="contacts__days">Saturday - Sunday</p>
-        <time class="contacts__time">08:00 Am</time> - <time class="contacts__time">16:00 pm</time>
+        <p class="contacts__title">Monday - Friday</p>
```

Comment:
Jeszcze dosłownie mała rzecz. W dniach tygodniach poprawnie zmieniłaś klasę,  aczkolwiek żeby było to bardzie spójne zmieniłabym w tych miejscach tag <p> na <h4>